### ISSUE TITLE
Troubles with set config in scripts

### ISSUE THREAD

**Issue by Barma-lej** [MTrab/landroid_cloud#33]
If i set config with script:
```
script:
  landroid_set_config:
    alias: "Landroid set config"
    sequence:
      service: landroid_cloud.config
      data_template:
        id: !secret landroid_id
        raindelay: "{{ states('input_number.landroid_raindelay') | int }}"
        timeextension: "{{ states('input_number.landroid_timeextension') | int }}"
```
then all values set to 0

So it works:
```
script:
  landroid_set_config:
    alias: "Landroid set config"
    sequence:
      service: landroid_cloud.config
      data_template:
        id: !secret landroid_id
        raindelay: 240
        timeextension: 10
```
Input Numbers:
```
input_number:
  landroid_raindelay:
    name: "Rain delay"
    initial: 150
    min: 0
    max: 300
    step: 30
    unit_of_measurement: "min"
    icon: "mdi:weather-lightning-rainy"

  landroid_timeextension:
    name: "Time extension"
    initial: 0
    min: -100
    max: 100
    step: 10
    unit_of_measurement: "%"
    icon: "mdi:clock-start"
```

**Comment by MTrab** [comment]
Try this:

```
script:
  landroid_set_config:
    alias: "Landroid set config"
    sequence:
      service: landroid_cloud.config
      data_template:
        id: !secret landroid_id
        raindelay: "{{ states('input_number.landroid_raindelay') }}"
        timeextension: "{{ states('input_number.landroid_timeextension') }}"
```

**Comment by Barma-lej** [comment]
Does not work  :(

**Comment by MTrab** [comment]
Okay - will test myself as it should work.

**Comment by MTrab** [comment]
Seems like Home Assistant is sending a string rather than an integer.
Will see if I can solve this in the integration for next release.

**Comment by MTrab** [comment]
Fixed in v1.6.1
